xAPI.md

```diff
@@ -1862,6 +1862,10 @@ The Experience API applies the concept of registration more broadly.  A registra
 considered to be an attempt, a session, or could span multiple Activities. There is no expectation that 
 completing an Activity ends a registration. Nor is a registration necessarily confined to a single Agent.
 
+The Registration id is also used when storing documents within the State resource, e.g. for 
```

Comment:
It's used in four other places in the spec, once in the parameters table for GET Statements and thrice in various parameters tables for the State resource. 

I'm not against removing it in all six places though since registration doesn't contain an object with an id property. Thoughts?